### PR TITLE
Add Containers topology populated by the Docker tagger.

### PR DESCRIPTION
**Pull request by tomwilkie**
Currently won't show in the UI, as there is no logic to use it.

Part of #171

### CODE REVIEW

File: report/report.go [hunk 1]
```diff
@@ -23,6 +23,11 @@ type Report struct {
 	// like operating system, load, etc. The information is scraped by the
 	// probes with each published report. Edges are not present.
 	Host Topology
+
+	// Container nodes represent all Docker containers on hosts running probes.
+	// Metadata includes things like Docker image, name etc.
+	// Edges are not present.
+	Container Topology
```
Comment (peterbourgon) [inline report/report.go]: Please stick it before Host

File: report/merge.go [hunk 2]
```diff
@@ -9,6 +9,7 @@ func (r *Report) Merge(other Report) {
 	r.Endpoint.Merge(other.Endpoint)
 	r.Address.Merge(other.Address)
 	r.Host.Merge(other.Host)
+	r.Container.Merge(other.Container)
```
Comment (peterbourgon) [inline report/merge.go]: Please stick it before Host

File: report/report.go [hunk 3]
```diff
@@ -67,9 +72,10 @@ type Row struct {
 // MakeReport makes a clean report, ready to Merge() other reports into.
 func MakeReport() Report {
 	return Report{
-		Endpoint: NewTopology(),
-		Address:  NewTopology(),
-		Host:     NewTopology(),
+		Endpoint:  NewTopology(),
+		Address:   NewTopology(),
+		Host:      NewTopology(),
+		Container: NewTopology(),
```
Comment (peterbourgon) [inline report/report.go]: Please stick it before Host

File: report/report.go [hunk 4]
```diff
@@ -80,6 +86,7 @@ func (r Report) Squash() Report {
 	r.Endpoint = r.Endpoint.Squash(EndpointIDAddresser, localNetworks)
 	r.Address = r.Address.Squash(AddressIDAddresser, localNetworks)
 	r.Host = r.Host.Squash(PanicIDAddresser, localNetworks)
+	r.Container = r.Container.Squash(PanicIDAddresser, localNetworks)
```
Comment (peterbourgon) [inline report/report.go]: Please stick it before Host

**Comment by tomwilkie** [comment]
@peterbourgon PTAL

**Comment by peterbourgon** [comment]
LGTM

**Pull request merged**